pkgs/top-level/all-packages.nix

```diff
@@ -17267,6 +17267,7 @@ with pkgs;
   cargo-binutils = callPackage ../development/tools/rust/cargo-binutils { };
   cargo-bloat = callPackage ../development/tools/rust/cargo-bloat { };
   cargo-bolero = callPackage ../development/tools/rust/cargo-bolero { };
+  cargo-bump = callPackage ../development/tools/rust/cargo-bump { };
```

Comment:
Could the package be moved to `pkgs/by-name`?

Reply:
My general understanding is that we shouldn't yet.

![2023-09-25_19-14](https://github.com/NixOS/nixpkgs/assets/89321978/c61ba4f9-16bd-4566-85d4-faeb44921a4d)

This is from yesterday.

Reply:
For existing packages, no. But this one is an init, right?

Reply:
Here's the [link](https://github.com/NixOS/nixpkgs/blob/f4822bb295ea194a1278af287771c615636cbcbf/pkgs/by-name/README.md#manual-migration-guidelines). It does seem to suggest that it should be initalized like that:

> This is the recommended way to add new top-level packages to Nixpkgs [when possible](https://github.com/NixOS/nixpkgs/blob/f4822bb295ea194a1278af287771c615636cbcbf/pkgs/by-name/README.md#limitations).

Do you want me to?

Reply:
Yes, please.

Reply:
I tried to use `by-name`, but seems like it will take a little extra work.

Reply:
Okay, works!